flow/datatypes/bigint.go

```diff
@@ -0,0 +1,12 @@
+package datatypes
+
+import (
+	"math/big"
+)
+
+func CountDigits(bi *big.Int) int {
```

Comment:
This seems like you found a bug in https://github.com/shopspring/decimal/pull/356 would be better to contribute fix upstream

Reply:
Any reason why string route isn't optimal?